test_handler.go

```diff
@@ -114,4 +86,26 @@ func GetRunSHA(r *http.Request) (runSHA string, err error) {
         runSHA = runParam
     }
     return runSHA, err
+}
+
+// GetBrowserNames loads, parses and returns the set of names of browsers
+// which are to be included (flagged as initially_loaded in the JSON).
+func GetBrowserNames() (browserNames []string, err error) {
```

Comment:
Should this be copied into memory exactly once? Do we expect the file system of an AppEngine instance to change underneath it without a restart?

Reply:
Probably not an important optimization (+ cost of development speed locally)

Reply:
> Do we expect the file system of an AppEngine instance to change underneath it without a restart?

Looks like AE deploys always restart the VM or create a new one.

> The shutdown process might be triggered by a variety of planned and unplanned events, such as:
> ...
> - You deploy an updated version to the service.

https://cloud.google.com/appengine/docs/standard/go/how-instances-are-managed

In local development too the server process is restarted after a file change.

Reply:
Added a TODO for keeping a static copy around.